pkg/exasol-rest-api/properties_file_parser.go

```diff
@@ -0,0 +1,45 @@
+package exasol_rest_api
+
+import (
+	"errors"
+	"gopkg.in/yaml.v3"
+	"os"
+)
+
+func getPropertiesFromFile(filepath string, properties *ApplicationProperties) error {
+	propertiesFile, err := openFile(filepath)
+	if err != nil {
+		return err
+	}
+	err = decodePropertiesFile(propertiesFile, properties)
+	if err != nil {
+		return err
+	}
+	closeFile(propertiesFile)
+	return nil
+}
+
+func openFile(filepath string) (*os.File, error) {
+	file, err := os.Open(filepath)
+	if err != nil {
+		return nil, err
+	} else if file == nil {
+		return nil, errors.New("properties file doesn't exist")
+	} else {
+		return file, nil
+	}
+}
+
+func decodePropertiesFile(propertiesFile *os.File, properties *ApplicationProperties) error {
```

Comment:
I think returning the `ApplicationProperties` would better express the purpose of this method

Reply:
refactored